clients/nasaClient.ts

```
@@ -0,0 +1,58 @@
+}
+
+interface PathParams {
+    rover: string;
+}
+
+interface QueryParams {
+    sol: number;
+    camera: string;
+}
+
+export const getRoverList = (): Promise<void | Rover[]> => {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)